docs/src/components/codeInputArea.tsx

```diff
@@ -0,0 +1,12 @@
+import clsx from 'clsx';
+import styles from '../pages/playground.module.css';
+
+const CodeInputArea = ({ onChange }) => (
```

Comment:
```suggestion
function PlaygroundCodeEditor({ onChange }) => (
```
also rename file to `PlaygroundCodeEditor` (the same applies to other components)

Not sure if this will work, you might also need to move function params to separate type.